guides/typography.md

```diff
@@ -1,146 +1,267 @@
+the ".kids-theme" CSS class is present. You can [see the theming guide for more guidance on defining
+multiple themes](https://material.angular.io/guide/theming#defining-multiple-themes).
 
 ```scss
-// Override the typography in the core CSS.
-@include mat-core($custom-typography-theme);
+@use '~@angular/material' as mat;
+
+@include mat.core();
+
+$my-primary: mat.define-palette(mat.$indigo-palette, 500);
+$my-accent: mat.define-palette(mat.$pink-palette, A200, A100, A400);
+
+$my-theme: mat.define-light-theme((
+ color: (
+   primary: $my-primary,
+   accent: $my-accent,
+ )
+));
+
+@include mat.all-component-themes($my-theme);
+
+.kids-theme {
+  $kids-primary: mat.define-palette(mat.$cyan-palette);
+  $kids-accent: mat.define-palette(mat.$yellow-palette);
+  $kids-typography: mat.define-typography-config(
+    // Specify "Comic Sans" as the default font family for all levels.
+    $font-family: 'Comic Sans',
+  );
+  
+  $kids-theme: mat.define-light-theme((
+   color: (
+     primary: $my-primary,
+     accent: $my-accent,
+   ),
+   typography: $kids-typography,
+  ));
+  
+  @include mat.all-component-themes($kids-theme);
+}
 ```
 
-For more details about the typography functions and default config, see the
-[source](https://github.com/angular/components/blob/master/src/material/core/typography/_typography.scss).
+Each component also has a `typography` mixin that emits only the typography styles for that
+component, based on a provided typography config. The following example demonstrates applying
+typography styles only for the button component.
 
+```scss
+@use '~@angular/material' as mat;
 
-## Material typography in your custom CSS
+$kids-typography: mat.define-typography-config(
+  // Specify "Comic Sans" as the default font family for all levels.
+  $font-family: 'Comic Sans',
+);
 
-Angular Material includes typography utility mixins and functions that you can use to customize your
-own components:
+// Now we have sweet buttons with Comic Sans.
+@include mat.button-typography($kids-typography);
+```
 
-* `mat-font-size($config, $level)` - Gets the `font-size`, based on the provided config and level.
-* `mat-font-family($config)` - Gets the `font-family`, based on the provided config.
-* `mat-line-height($config, $level)` - Gets the `line-height`, based on the provided
-config and level.
-* `mat-font-weight($config, $level)` - Gets the `font-weight`, based on the provided
-config and level.
-* `mat-typography-level-to-styles($config, $level)` - Mixin that takes in a configuration object
-and a typography level, and outputs a short-hand CSS `font` declaration.
+## Using typography styles in your application
+
+By default, the `core` and theming mixins only apply styles to Angular Material components. You
+can additionally add CSS classes for styling your application by including the
+`typography-hierarchy` mixin. This mixin emits CSS classes corresponding to each typography level.
```

Comment:
Not exactly-

`mat.all-component-themes` includes both `core-theme` and `all-component-typographies`. _However_, `all-component-themes` won't actually emit typography styles unless you actually specify a typography config. `all-component-typographies`, though, will _always_ emit typography styles no matter what you give it. 

In general, the idea is

`xxx-theme` === `xxx-color` + `xxx-typography` + `xxx-density` (mdc-only)

The same holds true for the "all" mixins:

`all-component-themes` === `all-component-colors` + `all-component-typographies` + `all-component-denstities` (still mdc-only)

"All" also includes core WRT `core-theme`. Also, `core-theme` is basically the same thing as `core-color` since there aren't any "core" typography or density styles.